Controllers/DatasetController.py

```diff
@@ -105,12 +105,35 @@ def sizeOfDataset(dataset_id):
 
     return Response(str(dataset.numRows), status=200) 
 
+@dataset.route("/addSize", methods=["GET"])
+def addSize():
+
+    #get all the datasets
+    datasets = Dataset.objects.filter()
+
+    #loop through them
+    for dataset in datasets:
+
+        print(dataset.numRows)
+        print(dataset.name)
+
+        if(dataset.name == "50K Sales"):
```

Comment:
I hardcoded it because when I was doing this there were two datasets that had the name 50k sales which messed things up. It was originally if(dataset.name != "50k Sales") but I changed it when I was trying to add the size to that dataset, but I don't think I actually did. The endpoint is not actually used by the website its just kind of a helper endpoint to ensure that all of the datasets had sizes.